examples/tutorials/workgraphs/descriptors_filter_qe.ipynb

```diff
@@ -0,0 +1,484 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5acf0b46",
+   "metadata": {},
+   "source": [
+    "### Setup\n",
+    "\n",
+    "The initial setup is very similar to the other tutorials, such as `singlepoint.ipynb`, which goes into more detail about what each step is doing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "73bfb65e",
+   "metadata": {},
+   "source": [
+    "Load the aiida profile and code:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "79098139",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiida import load_profile\n",
+    "load_profile()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3bc2cf3b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiida_mlip.data.model import ModelData\n",
+    "uri = \"https://github.com/stfc/janus-core/raw/main/tests/models/mace_mp_small.model\"\n",
+    "model = ModelData.from_uri(uri, architecture=\"mace_mp\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "806b9564",
+   "metadata": {},
+   "source": [
+    "We can load the computer and set the directory in which we want to store the data to. The work directory is a placeholder so change accordingly."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ae62816f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiida.orm import load_code, load_computer\n",
+    "\n",
+    "janus_code = load_code(\"janus@localhost\")\n",
+    "# qe_code = load_code(\"qe@scarf\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d661593",
+   "metadata": {},
+   "source": [
+    "We must now choose the calculations to perform:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0e4c78c9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aiida.plugins import CalculationFactory\n",
+    "geomoptCalc = CalculationFactory(\"mlip.opt\")\n",
+    "descriptorsCalc = CalculationFactory(\"mlip.descriptors\")\n",
+    "trainCalc = CalculationFactory(\"mlip.train\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "37df1f81",
+   "metadata": {},
+   "source": [
+    "Before setting up the work graph, we first configure the `Quantum Espresso (QE)` task by defining the code and input parameters. Since we need to run QE on multiple structures, we create multiple `PwCalculation` tasks dynamically within the same task using `get_current_graph()`. This allows us to run QE for each structure and return the corresponding `TrajectoryData` and parameters for each.\n",
+    "\n",
+    "**NOTE**: You will need `aiida-psuedo` installed to run this (available as an optional dependancy in `pyproject.toml`). After installing run the command: `aiida-pseudo install sssp` to install te required pseudopotential."
```

Comment:
We should probably have a section at the top with the assumptions, since, for example, we're assuming that the remote computer and code have been set up, etc.